RELEASE_NOTE_DRAFT.md

```diff
@@ -32,47 +102,47 @@ Acquiring Eclipse Collections
 <dependency>
   <groupId>org.eclipse.collections</groupId>
   <artifactId>eclipse-collections-api</artifactId>
-  <version>10.2.0</version>
+  <version>10.3.0</version>
 </dependency>
 
 <dependency>
   <groupId>org.eclipse.collections</groupId>
   <artifactId>eclipse-collections</artifactId>
-  <version>10.2.0</version>
+  <version>10.3.0</version>
 </dependency>
 
 <dependency>
   <groupId>org.eclipse.collections</groupId>
   <artifactId>eclipse-collections-testutils</artifactId>
-  <version>10.2.0</version>
+  <version>10.3.0</version>
   <scope>test</scope>
 </dependency>
 
 <dependency>
   <groupId>org.eclipse.collections</groupId>
   <artifactId>eclipse-collections-forkjoin</artifactId>
-  <version>10.2.0</version>
+  <version>10.3.0</version>
 </dependency>
 ```
 
 ### Gradle
 
 ```groovy
-compile 'org.eclipse.collections:eclipse-collections-api:10.2.0'
-compile 'org.eclipse.collections:eclipse-collections:10.2.0'
-testCompile 'org.eclipse.collections:eclipse-collections-testutils:10.2.0'
-compile 'org.eclipse.collections:eclipse-collections-forkjoin:10.2.0'
+compile 'org.eclipse.collections:eclipse-collections-api:10.3.0'
```

Comment:
Should this be `implementation` instead of `compile`?

Reply:
fixed